recipes/lantz/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "lantz" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # lantz-core
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-core/lantz-core-{{ version }}.tar.gz
+    sha256: 9ad1b74c1549fb31db492b3cb836bf710f4b7b9a0c729e0a908c7eec21db0afb
+  # lantz-dev
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}dev/lantzdev-0.5.2.tar.gz
```

Comment:
Why is this version different from core? Should they always be kept in sync? Or is it reasonable to mix and match versions of each component?

Reply:
@isuruf The codebase for the project is split between different repositories per the structural choices of the project owner. lantzdev packages all these repos together for publishing and has a different versioning number to denote changes in these packaging requirements.

Reply:
I think these should be separate conda packages instead of one conda package.

Reply:
I know it's not intuitive and this is actually something we are working to address on an upcoming release of the software. Lantzdev should really be rolled into Lantz-core as lantzdev is really just a shell. Similarly Lantz-core will not work without lantzdev. So they really shouldn't be packaged separated. I agree this versioning should be simplified and again this is something we are addressing in an upcoming release.